src/test/java/com/liferay/blade/cli/CreateCommandTest.java

```diff
@@ -40,18 +41,24 @@
 
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.StringUtils;
-
+import org.easymock.EasyMock;
 import org.gradle.testkit.runner.BuildTask;
 import org.gradle.tooling.internal.consumer.ConnectorServices;
 
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.easymock.PowerMock;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
 /**
  * @author Gregory Amerson
  */
+@PrepareForTest(CreateCommand.class)
```

Comment:
We can't use powermock runner for this test.  it is causing the server loader exceptions.